docker/serverless/Dockerfile.with_config

```diff
@@ -0,0 +1,7 @@
+# Serverless ESP docker image with build-in service config
+FROM _SERVERLESS_IMAGE
+
+ENV ENDPOINTS_SERVICE_PATH /etc/nginx/service.json
+ADD service.json ${ENDPOINTS_SERVICE_PATH}
```

Comment:
Should probably use `COPY` instead of `ADD`

https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy

Reply:
Done. Thanks